modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStoreManager.java

```diff
@@ -366,14 +366,22 @@ public FilePageStoreManager(GridKernalContext ctx) {
 
     /** {@inheritDoc} */
     @Override public void beginRecover() {
-        List<String> groupsWithWalDisabled = checkCachesWithDisabledWal();
+        List<CacheConfiguration> cacheCfgs = findCacheGroupsWithDisabledWal();
+
+        List<String> groupsWithWalDisabled = cacheCfgs.stream()
```

Comment:
Looks like this variable is no longer needed

Reply:
No, we still need this - the collection contains cache directories that later used in maintenance registry record.

Reply:
Fixed.